### PR TITLE
Sudo update fix for Ubuntu install instructions in branch 6.0.0

### PR DESCRIPTION
**Pull request by amd-jnovotny**
This adds the "sudo apt update" instruction to the Ubuntu install instructions in the ubuntu.rst file (in the Installation via Package Manager section). It also merges a fix from develop that corrects some variable names.

### CODE REVIEW

File: docs/how-to/native-install/post-install.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Post-installation instructions
     .. code-block:: bash
         :substitutions:
 
-        export PATH=$PATH:/opt/rocm-|rocm_version|/bin
+        export PATH=$PATH:/opt/rocm-|rocm_directory_version|/bin
```
Comment (yhuiYH) [inline docs/how-to/native-install/post-install.rst:28]: These substitutions are not taking effect.  There's probably something missing in conf.py like (line 60-62).

Like: ".. |amdgpu_version| replace:: {amdgpu_version}", but for "rocm_directory_version"

**Review by yhuiYH — CHANGES_REQUESTED** [review]
Needs some small changes, in conf.py, probably.

File: docs/how-to/native-install/post-install.rst (hunk 1 above)
Comment (amd-jnovotny) [reply docs/how-to/native-install/post-install.rst:28]: I made the change and it seems to be working now.

**Review by yhuiYH — APPROVED** [review]
LGTM now, please merge when ready.

**Pull request merged**